app/src/main/java/net/aiscope/gdd_app/ui/sample_completion/SampleCompletionViewModel.kt

```diff
@@ -146,6 +146,7 @@ class SampleCompletionViewModel @Inject constructor(
                     microscopistRepository.load().copy(hasSubmitSampleFirstTime = true)
                 )
             }
+            remoteStorage.storeSampleCollection(storedSample)

```

Comment:
Is this needed? Having it we will end up saving 2 samples each time, I think modifying the code just on top of this to accept the new Model will solve the issue.

Reply:
They end up in different locations though. This new 'storedsample' goes to https://console.firebase.google.com/u/0/project/aiscope-labelling-app-test/firestore/data/~2Fsamples~2F3a03bbd6-4047-4094-8c4a-3e6fa0863898

As opposed to https://console.firebase.google.com/u/0/project/aiscope-labelling-app-test/storage/aiscope-labelling-app-test.appspot.com/files/~2F3a03bbd6-4047-4094-8c4a-3e6fa0863898